man/kernel-install.xml

```diff
@@ -264,6 +264,14 @@
             <para>Some other layout not understood natively by <command>kernel-install</command>.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term>auto</term>
+          <listitem>
+            <para>Pick layout automatically.  This inspect the kernel image and <filename>$BOOT</filename>
```

Comment:
```suggestion
            <para>Pick the layout automatically.  This inspects the kernel image and <filename>$BOOT</filename>
```

Probably minor nit, the rest so the man pages doesn't do double spaces after periods, so this shouldn't either.